From cbecd86317c9f5b65f60f215523c38713d4c1c7c Mon Sep 17 00:00:00 2001
From: Alexey Khudyakov <alexey.skladnoy@gmail.com>
Date: Wed, 27 Nov 2019 12:08:10 +0300
Subject: [PATCH 2/2] Patch out building of ython bindings

It's funny: pybind11 is added as a submodule without pinning(!) thus nix fetch
latest masyter and breaks reproducibility
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e1c2084..bbf9963 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,6 +82,6 @@ if (EMSCRIPTEN)
   add_subdirectory(js-bindings)
 else()
   # emscripten can't build python bindings, it produces only javascript
-  add_subdirectory(contrib/pybind11)
-  add_subdirectory(python-bindings)
+  # add_subdirectory(contrib/pybind11)
+  # add_subdirectory(python-bindings)
 endif()
-- 
2.23.0

